docs/markdown/Contributing.md

```diff
@@ -76,6 +76,16 @@ notes. These features should be written in standalone files in the
 `docs/markdown/snippets` directory. The release manager will combine
 them into one page when doing the release.
 
+### Skipping integration testing
+
+Integration tests are expensive and not necessary for documentation changes.
+Tests can be disabled by putting `[skip ci]` or `[ci skip]` into commit title.
+
+- [Travis-CI](https://docs.travis-ci.com/user/customizing-the-build/#Skipping-a-build)
+  allows `[skip ci]` anywhere in the commit messages.
+- [AppVeyor](https://www.appveyor.com/docs/how-to/filtering-commits/#skip-directive-in-commit-message)
+  requires `[skip ci]` in the commit title.
```

Comment:
Might be useful to mention that appveyor can be skipped by putting `[skip appveyor]` in the title. This is sometimes useful when trying to debug a failure on Linux or macOS, or changes to a test that won't run on Windows.